apps/docs/docs/faq.md

```diff
@@ -25,3 +25,69 @@ This is a good option if you are using TypeScript/JavaScript and want to interac
 - [Boilerplate](https://github.com/bandada-infra/boilerplate)
 
 This is a good option if you want to quickly create a Bandada project because you can fork it, clone it or use it as a template.
+
+## What is the difference between Bandada and Semaphore?
+
+When it comes to [Bandada](https://bandada.pse.dev/) and [Semaphore](https://semaphore.pse.dev/), you might find yourself wondering how these two platforms stack up against each other. Both are designed to streamline and enhance different aspects of project management and automation, but they cater to distinct needs and offer unique features. Here’s a comprehensive comparison to help you understand how they differ.
```

Comment:
Semaphore is not a platform, it's a project or protocol.

Neither project is focused on project management or automation.